RLTSquare/Unit3/Block/MultiWebsite.php

```diff
@@ -0,0 +1,89 @@
+<?php
+declare(strict_types=1);
+
+namespace RLTSquare\Unit3\Block;
+
+use Magento\Framework\Encryption\EncryptorInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\View\Element\Template\Context;
+use RLTSquare\Unit3\Helper\MultiWebsiteConfiguration;
+use RLTSquare\Unit3\Model\Config\Source\Types;
+
+/**
+ * block class for getting configuration values
+ */
+class MultiWebsite extends \Magento\Framework\View\Element\Template
+{
+    /**
+     * @var EncryptorInterface
+     */
+    protected EncryptorInterface $decryptor;
+
+    /**
+     * @var MultiWebsiteConfiguration
+     */
+    protected MultiWebsiteConfiguration $helper;
+    /**
+     * @var Types
+     */
+    protected Types $types;
+
+    /**
+     * @param Context $context
+     * @param MultiWebsiteConfiguration $helper
+     * @param Types $types
+     * @param EncryptorInterface $decryptor
+     * @param array $data
+     */
+    public function __construct(
+        Context                   $context,
+        MultiWebsiteConfiguration $helper,
+        Types                     $types,
+        EncryptorInterface        $decryptor,
+        array                     $data = []
+    ) {
+        $this->helper = $helper;
+        $this->types = $types;
+        $this->decryptor = $decryptor;
+        parent::__construct($context, $data);
+    }
+
+    /**
+     * @throws NoSuchEntityException
+     */
+    public function getEnable()
+    {
+        return $this->helper->getEnable();
+    }
+
+    /**
+     * @throws NoSuchEntityException
+     */
+    public function getUsername()
+    {
+        return $this->helper->getUserName();
+    }
+
+    /**
+     * @throws NoSuchEntityException
+     */
+    public function getPassword(): string
```

Comment:
Just a thought, not a deal braker ... Might be nice to use `$decrypted=true` parameter here; which default to true, but can be set to false, so the function returns either decrypted or encrypted pass when/if you need it.